ocs_ci/deployment/deployment.py

```diff
@@ -649,6 +649,25 @@ def deploy_ocs_via_operator(self, image=None):
                 "enable": True,
             }
 
+        if config.DEPLOYMENT["ceph_debug"]:
+            ceph_debug_log_configmap_data = templating.load_yaml(
+                constants.CEPH_CONFIG_DEBUG_LOG_LEVEL_CONFIGMAP
+            )
+            ceph_configmap_yaml = tempfile.NamedTemporaryFile(
+                mode="w+", prefix="config_map", delete=False
+            )
+            templating.dump_data_to_temp_yaml(
+                ceph_debug_log_configmap_data, ceph_configmap_yaml.name
+            )
+            logger.info(
+                "Setting Ceph to work in debug log level using a new configmap resource"
+            )
+            run_cmd(f"oc create -f {ceph_configmap_yaml.name}", timeout=1200)
+
+            cluster_data["spec"]["managedResources"] = {
+                "cephConfig": {"reconcileStrategy": "ignore"}
+            }
```

Comment:
The `deploy_ocs_via_operator` function is already quite large (and is growing). Can we move this logic to another function and call it from here? Something like `setup_ceph_debug` maybe?

Reply:
yes, will do